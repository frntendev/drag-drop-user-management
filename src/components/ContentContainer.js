import React from "react";
import styled from "react-emotion";
const ContentContainer = styled("div")`
  /* width: ${props => props.gsize && props.gsize + "px"}; */
  ${props => props.customStyle};
  border-radius:2px;
  padding-left: 24px;
  padding-right: 24px;
  padding-bottom: 16px;
  background-color: ${props => props.bcolor};
  box-shadow: ${props =>
    props.shadow &&
    "0px 3px 5px -1px rgba(0, 0, 0, 0.2),0px 6px 10px 0px rgba(0, 0, 0, 0.09),0px 1px 18px 0px rgba(0, 0, 0, 0.06)"};
  @media (max-width: 800px) {
    padding: 0;
    width: 100%;
    margin-top: 5%;
  }
`;
export default props => {
  return <ContentContainer {...props} />;
};
