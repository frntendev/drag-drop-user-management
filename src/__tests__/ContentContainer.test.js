import React from "react";
import { shallow, mount, configure } from "enzyme";
import ContentContainer from "../components/ContentContainer";
import Adapter from "enzyme-adapter-react-15";
configure({ adapter: new Adapter() });
describe("ExampleComponent", () => {
  test("renders component itself correctly", () => {
    const renderedComponent = shallow(<ContentContainer></ContentContainer>);
    expect(renderedComponent).toMatchSnapshot();
  });
  test("renders compoenent and children correctly",()=>{
      const renderedComponent = shallow(<ContentContainer><div className="test-classname">Test</div></ContentContainer>)
      expect(renderedComponent).toMatchSnapshot();
  })
});
