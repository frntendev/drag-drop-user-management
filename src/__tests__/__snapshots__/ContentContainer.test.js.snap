// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExampleComponent renders compoenent and children correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown>
    <div
      className="test-classname"
    >
      Test
    </div>
  </Unknown>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <div
        className="test-classname"
      >
        Test
      </div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Test",
        "className": "test-classname",
      },
      "ref": null,
      "rendered": "Test",
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <div
          className="test-classname"
        >
          Test
        </div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Test",
          "className": "test-classname",
        },
        "ref": null,
        "rendered": "Test",
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;

exports[`ExampleComponent renders component itself correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {},
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
